tests/no-explicit-type-exports.test.ts

```diff
@@ -305,17 +309,6 @@ ruleTester.run('no-explicit-type-exports', rule, {
         },
       ],
     },
-    {
-      // The rule fails when export and import a type or interface on a single line
-      filename: fileName,
-      code: "import { x } from './oneLine'; export type { x };",
-      output: "import type { x } from './oneLine';\n export type { x };",
-      errors: [
-        {
-          message: "Do not export 'x' it is an imported type or interface.",
-        },
-      ],
-    },
```

Comment:
Now we allow import  and export of a type so this test started failing. This test was passing as invalid because of the type export, not because of the one-liner (I think it was a miss), but now this is fixed and all works as expected. The other one-liner import/export pass too (see full file)